packages/driver/test/cypress/integration/e2e/focus_blur_spec.js

```diff
@@ -650,7 +687,11 @@ describe('intercept blur methods correctly', () => {
       <img usemap="#map" src="/__cypress/static/favicon.ico" alt="image" />
       `).appendTo(cy.$$('body'))
 
-      cy.get('area').focus().should('have.focus')
+      cy.get('area')
+      // NOTE: wait needed for firefox, otherwise element is not yet ready/loaded
```

Comment:
this is a good message and what i'd expect anytime your logic has a fork that's not clear